tests/test_pregel.py

```diff
@@ -1891,7 +1891,6 @@ def search_api(query: str) -> str:
     assert app.get_input_schema().schema_json() == snapshot
     assert app.get_output_schema().schema_json() == snapshot
     assert json.dumps(app.get_graph().to_json(), indent=2) == snapshot
-    assert app.get_graph().draw_ascii() == snapshot
```

Comment:
I'm finding that this test is flaky, lmk if it should be deterministic.

https://github.com/langchain-ai/langgraph/pull/319/commits/bac26213e949a78c10606a1b2c96faf6cb964f80